test/unittests/commands/xmlupload/test_ontology_problem_models.py

```diff
@@ -1,88 +1,149 @@
 import pandas as pd
 from pandas.testing import assert_frame_equal
 
-from dsp_tools.commands.xmlupload.models.ontology_problem_models import InvalidOntologyElementsInData
+from dsp_tools.commands.xmlupload.models.ontology_lookup_models import TextValueData
+from dsp_tools.commands.xmlupload.models.ontology_problem_models import (
+    InvalidOntologyElementsInData,
+    InvalidTextValueEncodings,
+)
 
 
-def test_print_problem_string_cls() -> None:
-    onto = InvalidOntologyElementsInData([("clsA", ["idA"], "wrong")], [], ontos_on_server=["test"])
-    msg = onto._compose_problem_string_for_cls()
-    assert msg == (
-        "The following resource(s) have an invalid resource type:\n\n"
-        "    Resource Type: 'clsA'\n"
-        "    Problem: 'wrong'\n"
-        "    Resource ID(s):\n"
-        "    - idA"
-    )
+class TestInvalidOntologyElementsInData:
```

Comment:
I did not change any tests, just put them in a class but ruff seems to have reformatted a lot of things.